eslint.config.mjs

```diff
@@ -16,6 +16,13 @@ export default [
   pluginJs.configs.recommended,
   ...tseslint.configs.recommended,
   eslintConfigPrettier, // > Turns off all rules that are unnecessary or might conflict with Prettier.
+  {
+      // The Redocly plugin uses CommonJS syntax for imports and exports as in the Redocly documentation.
+      files: ["model-server-openapi/**/*"],
+      rules: {
+        "@typescript-eslint/no-require-imports": "off"
+      }
+  },
```

Comment:
Maybe this change could be part of the first commit?

Reply:
Done